ONSdigital/eq-survey-runner
# eq-534 Change 'from/to' labels to 'period from/to'

### What is the context of this PR?

Following accessibility testing, it was recommended to provide more context in the "from/to" labels in the date range question by changing them to "period from" and "period to". This will provide a better experience for users of assistive software.
### How to review
1. Start a survey
2. Confirm label has changed.
3. Evoke validation message by providing an incorrect date range whereby the "period to" date is 
   prior to or the same as the "period from" date or similar)
4. Confirm updated error message makes sense
